.travis.yml

```diff
@@ -61,7 +61,24 @@ script:
   - java -version
   - if [ -n "$CLANG_FORMAT_PATH" ]; then bin/clangfmt --test; fi
   - bin/scalafmt --test
-  - bash sbt -J-Xmx2G rebuild "set scriptedBufferLog in sbtScalaNative := false" test-all
+  - sbt -J-Xmx2G rebuild
+  - mkdir utest
+  - pushd utest
+  - git init .
+  - git remote add -t cross-native Duhemm https://github.com/Duhemm/utest.git
+  - git fetch Duhemm cross-native
+  - git checkout Duhemm/cross-native
+  - sbt publishLocal
+  - popd
+  - mkdir scalacheck
+  - pushd scalacheck
+  - git init .
+  - git remote add -t cross-native Duhemm https://github.com/Duhemm/scalacheck.git
+  - git fetch Duhemm cross-native
+  - git checkout Duhemm/cross-native
+  - sbt publishLocal
+  - popd
+  - bash sbt -J-Xmx2G "set scriptedBufferLog in sbtScalaNative := false" sandbox/test test-all
```

Comment:
As soon as we remove temporary dependency on your branch of `utest` and `scalacheck`, we'll have no tests whatsoever for the testing framework integration. I suggest that we migrate our current mini-framework we used for `tests` project to the testing framework integration to address this.